.github/workflows/lint.yaml

```
@@ -26,6 +27,8 @@ jobs:
           base: ${{ github.event.pull_request.base.ref || github.event.merge_group.base_ref }}
           ref: ${{ github.event.pull_request.head.ref || github.event.merge_group.head_ref }}
           filters: |
+            has_rfd:
+              - 'rfd/**'
```

```suggestion
              - 'rfd/**'
              - '.github/workflows/lint.yaml'
```

This workflow should be included as well so that changes to the workflow are validated at PR time